std_msgs/Header header

bool driver_blinker_bulb_on          # True if ON
bool passenger_blinker_bulb_on       # True if ON
bool driver_blinker_bulb_on_avail    # True if this information is available on the platform.
                                     # Does not change during operation.
bool passenger_blinker_bulb_on_avail # True if this information is available on the platform.
                                     # Does not change during operation.
